binary_transparency/firmware/internal/verify/bundle.go

```diff
@@ -60,7 +69,11 @@ func verifyBundle(b api.ProofBundle) (api.FirmwareMetadata, error) {
 	if err := json.Unmarshal(b.ManifestStatement, &fwStatement); err != nil {
 		return api.FirmwareMetadata{}, fmt.Errorf("failed to unmarshal FirmwareStatement: %w", err)
 	}
-	// TODO(al): check sig on fwStatement
+	// Verify the signature:
```

Comment:
`// Verify the statement signature` - to be clear (particularly since we'll add sigs on checkpoints later on).